### DISCUSSION TITLE
Ошибка при использовании byedpi как Системный прокси

### DISCUSSION THREAD

**Discussion by ropucyka** [hufrea/byedpi#103] [Q&A]
Windows 10. Прописал в настройках прокси- адрес 127.0.0.1 и порт который слушает byedpi.
При попытках браузера или какой-либо программы подключиться к сети, byedpi сыпет такие сообщения:

![2024-08-24_235723](https://github.com/user-attachments/assets/a20a68be-dbaa-4c83-b601-a963c80f35c0) ![2024-08-25_000313](https://github.com/user-attachments/assets/36a1f7ea-8eba-45e5-bf47-21d7a80641da)

**http**://translate.google.ru
![2024-08-24_235602](https://github.com/user-attachments/assets/fe66c081-136d-41ca-9f43-778e44f80537)

**https**://translate.google.ru
![2024-08-24_235608](https://github.com/user-attachments/assets/c086c03c-59be-47de-a472-9809b47bc6fd)

**Comment by lufog** [comment]
Уверены, что правильно указали IP? Если указываете через приложение Настройки, перед ip нужно добавить `socks=` например `socks=127.0.0.1`.
https://github.com/hufrea/byedpi/issues/32#issuecomment-2270688476

**Comment by ropucyka** [comment]
Спасибо. Вопрос решён. Работает.

Указал через классические настройки Win+R - inetcpl.cpl

![2024-08-25_011353](https://github.com/user-attachments/assets/52f0bba4-4149-4e4e-a6c6-94e48e4f1200) ![2024-08-25_011357](https://github.com/user-attachments/assets/e2224b9d-6629-4f56-99a4-ce3d0e1215bf)
![2024-08-25_011407](https://github.com/user-attachments/assets/be510ecc-3d1f-4486-ad7c-a917f0d9c9a2)
